main.go

```diff
@@ -16,6 +16,57 @@ import (
 	"github.com/bacalhau-project/bacalhau/pkg/logger"
 )
 
+//	@title			Bacalhau API
+//	@version		v1
+//	@description	The Bacalhau API is a RESTful API that allows you to interact with the Bacalhau network.
+//	@termsOfService	http://bacalhau.org/terms/
+
+// TODO: #3165 Host the terms of service on bacalhau.org/terms
+
+//	@contact.name	API Support
+//	@contact.url	https://www.expanso.io/contact/
+//	@contact.email	support@bacalhau.org
+
+// TODO: #3166 Host an email address and a contact form on bacalhau.org/support
+
+//	@license.name	Apache 2.0
+//	@license.url	http://www.apache.org/licenses/LICENSE-2.0.html
+
+//	@host		localhost:1234
+//	@BasePath	/api/v1
+
+// @externalDocs.description 	Bacalhau Documentation
+// @externalDocs.url			https://docs.bacalhau.org
+
+////	@securityDefinitions.basic	BasicAuth
+
+////	@securityDefinitions.apikey	ApiKeyAuth
+////	@in							header
+////	@name						Authorization
+////	@description				Description for what is this security definition being used
+
+// -- Add authentication to swagger here
+//// 	@securitydefinitions.oauth2.application	OAuth2Application
+//// 	@tokenUrl								https://example.com/oauth/token
+//// 	@scope.write							Grants write access
+//// 	@scope.admin							Grants read and write access to administrative information
+//
+//// 	@securitydefinitions.oauth2.implicit	OAuth2Implicit
+//// 	@authorizationUrl						https://example.com/oauth/authorize
+//// 	@scope.write							Grants write access
+//// 	@scope.admin							Grants read and write access to administrative information
+//
+//// 	@securitydefinitions.oauth2.password	OAuth2Password
+//// 	@tokenUrl								https://example.com/oauth/token
+//// 	@scope.read								Grants read access
+//// 	@scope.write							Grants write access
+//// 	@scope.admin							Grants read and write access to administrative information
+//
+//// 	@securitydefinitions.oauth2.accessCode	OAuth2AccessCode
+//// 	@tokenUrl								https://example.com/oauth/token
+//// 	@authorizationUrl						https://example.com/oauth/authorize
+//// 	@scope.admin							Grants read and write access to administrative information
+
```

Comment:
These still refer to example domains and we don't use OAuth2 as standard (yet)

Reply:
just to be clear, these are all commented out (4 '/'s) - just placeholders for the future.